src/index.js

```diff
@@ -250,7 +250,15 @@ class Firestore {
      * @private
      * @type {Object}
      */
-    this._initalizationOptions = options;
+    this._initalizationOptions = settings;
```

Comment:
Should this be renamed to _initializationSettings for consistency?

Reply:
That makes sense. Done.